src/schemas/base-types.json

```diff
@@ -42,7 +42,7 @@
 	"uint": {
 		"title": "hex encoded unsigned integer",
 		"type": "string",
-		"pattern": "^0x([1-9a-f]+[0-9a-f]*|0)$"
+		"pattern": "^0x[0-9a-f]+$"
```

Comment:
I don't think this is correct simplification as for `uint` type no trailing zeros allowed. For example:
* incorrect: `0x0f`
* correct: `0xf`

Reply:
Though, `0x0` is correct, i.e. trailing zeros allowed only in case when value is `0`